### ISSUE TITLE
Support Emotion

### ISSUE THREAD

**Issue by deadcoder0904** [styled-components/vscode-styled-components#107]
I think Emotion with Template Literal is same as Styled Components so if it can be supported, it would be great for many people :)

**Comment by erickrawczyk** [comment]
@deadcoder0904 I'm not having any issues using this package with emotion. are you using emotion's `styled` import?

**Comment by deadcoder0904** [comment]
Yes

**Comment by erickrawczyk** [comment]
@deadcoder0904 Ah, I'm using the `react-emotion` package; not sure if you're using that or not

![image](https://user-images.githubusercontent.com/10940268/42532145-f815a59e-8453-11e8-9ad4-50df9cddba9f.png)

**Comment by Kovensky** [comment]
An example where specifically emotion is not supported is when the second argument to `styled` is given. Compare:

![image](https://user-images.githubusercontent.com/73085/44201264-c7d1c000-a183-11e8-8388-272016c84a6a.png)

![image](https://user-images.githubusercontent.com/73085/44201304-e3d56180-a183-11e8-9b4a-de48da621cbd.png)

styled-components only takes a single argument for the `styled` call.

**Comment by deadcoder0904** [comment]
Uhh ohh Max its already supported 😂

Don't know what I was thinking when I opened up the issue. I guess it wasn't working then.

**Comment by mxstbr** [comment]
Yeah the basic `styled.bla` API is, we don't differentiate whether you import from styled-components or emotion, that working is super cool! What I'm saying is that we won't support any emotion-specific API surface area like the second argument mentioned above or anything else.

**Comment by Kovensky** [comment]
It would be great to get that working too but I'm not sure it can be made to work with just regular expressions; a parser plugin would probably be needed.

**Comment by mxstbr** [comment]
Yeah that's what I mean: I don't want us to have to maintain any code for emotion-specific support even though we don't even use it, but as I said you're more than welcome to fork this package
